static/templates/viewer_settings.js

```diff
@@ -0,0 +1 @@
+const linkBlockingEnabled = {{enable_link_blocking}};
```

Comment:
Wouldn't be better to move this in a `setting` object ?
It would avoid introduction of magic global variables which are always to know from where they come from few months later.

And would it worth to move this in json ?

Reply:
> Wouldn't be better to move this in a `setting` object ?

Indeed. Done.

> And would it worth to move this in json ?

I can see disadvantages (will have to load the json with a `fetch()` vs the browser loading the .js file automatically) but no advantages.

Reply:
> will have to load the json with a fetch() vs the browser loading the .js file automatically)

Indeed. If we need a full API for a js application loading the setting, we will be able to do it when needed